templates/daily_location.html

```diff
@@ -95,78 +121,80 @@
-                    <option value="21">9 PM</option>
-                    <option value="22">10 PM</option>
-                    <option value="23">11 PM</option>
-                    <option value="24" selected>12 PM</option>
+                <select id="end-time" name="endtime" onchange="on_select_change()">
+                    <option value="3600">1 AM</option>
+                    <option value="7200">2 AM</option>
+                    <option value="10800">3 AM</option>
+                    <option value="14400">4 AM</option>
+                    <option value="18000">5 AM</option>
+                    <option value="21600">6 AM</option>
+                    <option value="25200">7 AM</option>
+                    <option value="28800">8 AM</option>
+                    <option value="32400">9 AM</option>
+                    <option value="36000">10 AM</option>
+                    <option value="39600">11 AM</option>
+                    <option value="43200">12 PM</option>
+                    <option value="46800">1 PM</option>
+                    <option value="50400">2 PM</option>
+                    <option value="54000">3 PM</option>
+                    <option value="57600">4 PM</option>
+                    <option value="61200">5 PM</option>
+                    <option value="64800">6 PM</option>
+                    <option value="68400">7 PM</option>
+                    <option value="72000">8 PM</option>
+                    <option value="75600">9 PM</option>
+                    <option value="79200">10 PM</option>
+                    <option value="82800">11 PM</option>
+                    <option value="86400">11:59 PM</option>
                 </select>
-            </div>
+            </form>
+        </div>
 
         <!--The div element for the map -->
-        <div id="map" style="width:99%; margin:auto; height:95%"></div>
-            <!-- JS for Firebase -->
-            <!-- The core Firebase JS SDK is always required and must be listed first -->
-            <script src="https://www.gstatic.com/firebasejs/7.8.0/firebase-app.js"></script>
-            <!-- Realtime database -->
-            <script src="https://www.gstatic.com/firebasejs/7.8.0/firebase-database.js"></script>
+        <div id="map"></div>
+
+        <!-- Hidden by default -->
+        <div id="no-info">
+            <h3>No Activity For Selected Period</h3>
+        </div>
 
-            <!-- Initialize map -->
-            <!-- My Script -->
-            <script src="../static/map.js"></script>
+        <!-- Map script-->
+        <script src="../static/map.js"></script>
 
-            <!-- JS to initialize map -->
-            <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCAI0alGN1v2HEMQvKXrXoWQJ9G9jRIHYI&callback=initMap"></script>
+        <!-- Initialize map -->
+        <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDhSAsZyMmmRGxnH7g_BnV6jrU7Xc30goI&callback=draw_map"></script>
```

Comment:
The API key should not be stored within the public GitHub repository. Instead it should be in a file that is in .gitignore like the Firebase key is.